### ISSUE TITLE
Error when saving record: unhashable type: 'list'

### ISSUE THREAD

**Issue by pbelouin-mpiwg** [bhch/django-jsonform#112]
Hello,

Thanks for the great library! A new error appeared on my end, and I can't figure it out! It was working previously, and all of the sudden I end up with this issue when trying to save a record using django admin with Django_jsonform:

```
File "/usr/local/lib/python3.10/site-packages/django_jsonform/utils.py", line 40, in normalize_keyword
return normalize_keyword.kw_map.get(kw, kw)
TypeError: unhashable type: 'list'
```
I have tried to change my schema to pinpoint where the issue was, to no avail. Is this something that you have run into? I'm quite stumped and would really appreciate pointers on that one!

Here is my schema - I've checked it with a validator, and it seems fine as far as I can see...

```
{
  "type": "object",
  "additionalProperties": true,
  "properties": {
    "combio": {
      "type": "object",
      "properties": {
        "participants": {
          "type": "array",
          "items": {
            "type": "object",
            "properties": {
              "name": {
                "type": "string"
              },
              "role": {
                "type": "string",
                "enum": ["interviewer", "interviewee", "participant"]
              }
            }
          }
        },
        "title": {
          "type": "string"
        },
        "permalink": {
          "type": "string"
        },
        "transcript": {
          "type": "string"
        },
        "collection": {
          "type": "string"
        },
        "date": {
          "type": ["string", "null"],
          "format": "date"
        }
      },
      "required": ["title", "collection", "permalink"]
    },
    "dc": {
      "type": "object",
      "properties": {
        "description": {
          "type": "string"
        },
        "publisher": {
          "type": ["string", "null"]
        }
      }
    }
  },
  "title": "combio_metadata_scheme"
}
```

**Comment by bhch** [comment]
What version are you on? Check using:

```sh
$ python -c "import django_jsonform; print(django_jsonform.__version__)"
```

**Comment by pbelouin-mpiwg** [comment]
currently rebuilding my docker stack so this might take a while, but from my poetry.lock file it seems to be 2.16.0

**Comment by bhch** [comment]
Ok no worries. I just tested it with the latest version and I'm getting this error as well.

**Comment by pbelouin-mpiwg** [comment]
oh interesting, trying with my schema you mean? thank you so much for your responsiveness by the way really appreciate it

**Comment by bhch** [comment]
Supplying multiple types is not supported yet. This is what's causing the problem: `"type": ["string", "null"]`.

**Comment by pbelouin-mpiwg** [comment]
I confirm I'm on 2.16.0, using 
```
$ python -c "import django_jsonform; print(django_jsonform.__version__)"
```

**Comment by pbelouin-mpiwg** [comment]
> Supplying multiple types is not supported yet. This is what's causing the problem: `"type": ["string", "null"]`.

I see! I did introduce this change recently to allow for empty publishers - thank you so much, I'll change it back and hopefully that should do it then!

**Comment by bhch** [comment]
I can release a quick fix for this bug if you don't want to change the schema.

The widget will handle this case more gracefully without crashing. But multiple types will still not be fully supported.

**Comment by pbelouin-mpiwg** [comment]
well, I actually use the schema for validation and other things as well, so that would be certainly be a welcome fix! I've changed the schema and it works now, so as far as I'm concerned it's not urgent however 👍

Strangely as well looks like, on my end, poetry didn't pick up the latest versions you pushed, but this might stem from my lack of experience with poetry...

**Comment by bhch** [comment]
Temporary fix released in [v2.17.4](https://github.com/bhch/django-jsonform/releases/tag/v2.17.4). Please upgrade and test it out.

Support for multiple types shall be added in future. I've created #113 to track this.

Thanks for reporting this issue.
